flows/automation-assessments.py

```diff
@@ -0,0 +1,298 @@
+import asyncio
+import os
+from contextlib import asynccontextmanager
+from datetime import timedelta
+from typing import Any, AsyncGenerator, Dict
+from uuid import uuid4
+
+import anyio
+import pendulum
+from packaging.version import Version
+
+import prefect
+
+TEST_SERVER_VERSION = os.environ.get("TEST_SERVER_VERSION", prefect.__version__)
+
+if Version(TEST_SERVER_VERSION) < Version("2.17.2"):
```

Comment:
In a bitter irony, this won't work until we publish the next version of Prefect because it requires client and server changes and we run these tests with the published images.